templates/overrides/search/form-element--sort-by-bef.html.twig

```diff
@@ -0,0 +1,36 @@
+{#
```

Comment:
remove this file, do it from oe_whitelabel_float_end_sort, you can controle the entire sort form from there, this way the view gets the theming only when the plugin is selected